### ISSUE TITLE
Switch issue template from using check boxes to tables the same as in microsoft-ui-xaml repository

### ISSUE THREAD

**Issue by xperiandri** [unoplatform/uno#6833]
## What would you like to be added:
See https://github.com/microsoft/microsoft-ui-xaml/issues/5402 as example

## Why is this needed:
Checkboxes were designed to track work progress. So they make visual noise on the issue list.

**Comment by Youssef1313** [comment]
Actually GitHub recently introduced a new feature for issue templates known as **[issue forms](https://docs.github.com/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)** (currently in beta). We might use that as well. See [example](https://github.com/dotnet/docs/issues/new?assignees=gewarren&labels=breaking-change%2CPri1%2Cdoc-idea&template=breaking-change.yml&title=%5BBreaking+change%5D%3A+) from dotnet/docs repo.

**Comment by jeromelaban** [comment]
Yes forms are going to be a better use of this. Tables need a bit too much editing work, and check boxes are not tasks indeed. We'll investigate forms now that it's available.
